src/Native/Runtime/gcrhinterface.h

```diff
@@ -166,6 +168,10 @@ class RedhawkGCInterface
     // to emit allocation ETW events with type information.  We set this value unconditionally to avoid
     // race conditions where ETW is enabled after the value is set.
     DECLSPEC_THREAD static EEType * tls_pLastAllocationEEType;
+
+    // Tracks the amount of bytes that were reserved for threads in their gc_alloc_context and went unused when they died.
+    // Used for GC.GetTotalAllocatedBytes
+    static uint64_t deadThreadsNonAllocBytes;
```

Comment:
Do you prefer PascalCase or camelCase? With or without `m_`? I'm unsure about the conventions in CoreRT

Reply:
The prevalent convention for statics is `s_` prefix (same as what the convention is for CoreCLR/CoreFX C# code), but it is not 100% consistent.